rustls/src/anchors.rs

```diff
@@ -34,15 +25,13 @@ impl OwnedTrustAnchor {
     }
 }
 
-impl From<webpki::TrustAnchor<'_>> for OwnedTrustAnchor {
-    fn from(t: webpki::TrustAnchor) -> OwnedTrustAnchor {
-        Self::from_trust_anchor(&t)
-    }
-}
-
-impl<'a> From<&'a OwnedTrustAnchor> for webpki::TrustAnchor<'a> {
-    fn from(anchor: &'a OwnedTrustAnchor) -> Self {
-        anchor.to_trust_anchor()
+impl From<&webpki::TrustAnchor<'_>> for OwnedTrustAnchor {
+    fn from(t: &webpki::TrustAnchor) -> OwnedTrustAnchor {
+        OwnedTrustAnchor {
```

Comment:
Self.

I think there's a clippy lint that will do this style check for us. We should enable it.